src/test/java/testsuite/regression/ConnectionRegressionTest.java

```diff
@@ -11565,7 +11520,7 @@ public void testBug98445() throws Exception {
         System.setErr(new PrintStream(newErr));
 
         // clientInfoProvider=CommentClientInfoProvider
-        testConn1 = getConnectionWithProps("clientInfoProvider=CommentClientInfoProvider,profileSQL=true");
+        testConn1 = getConnectionWithProps("clientInfoProvider=CommentClientInfoProvider,profileSQL=true,logger=StandardLogger");
```

Comment:
why do we need to setup a looger?

Reply:
this test uses the aws protocol, and with this pr logging is off by default for the aws protocol. As far as I can tell the test relies on logging information being spit out (see the assertion on line 11532). This assertion started failing when I turned the logging off and passes when I add this connection property to turn it back on